sql/database.sql

```diff
@@ -10610,13 +10611,14 @@ CREATE TABLE `login_mfa_registrations` (
 DROP TABLE IF EXISTS `api_token`;
 CREATE TABLE `api_token` (
     `id`           bigint(20) NOT NULL AUTO_INCREMENT,
-    `token_api`    varchar(40),
+    `token_api`    varchar(4),
     `user_id`      bigint(20) NOT NULL,
     `patient_id`   bigint(20) NOT NULL,
```

Comment:
@bradymiller 
I was thinking about this table last night and it occured to me, why do we need to maintain a history of expired tokens?

Reply:
@sjpadgett We shouldn't be doing that 😂 our database is big enough as it is already

Reply:
we already do maintain expired tokens! And for mysql, our database is puny!!!

Reply:
@sjpadgett , I suppose keeping them around is helpful for auditting/logging. We could definitely place a DELETE query in the isValidToken() function that removes all expired tokens (or maybe been expired for a day). Just let me know if you want it and I'll add it with this PR.

Reply:
I was thinking the same as the only reason however we'd not know what user was upto so really no reason to carry the load.
I'd do a insert/replace or delete query and not have another ongoing large table for admins to clean up 2 years down the road.